Cargo.toml

```diff
@@ -40,27 +39,29 @@ flate2 = "1"
 http = "0.2"
 once_cell = "1"
 futures-util = { version = "0.3", default-features = false, optional = true }
-opentelemetry = "0.21"
-opentelemetry_sdk = "0.21"
-opentelemetry-http = "0.10"
-opentelemetry-semantic-conventions = "0.13"
+opentelemetry = "0.22"
+opentelemetry_sdk = "0.22.1"
```

Comment:
if this is just "0.22", won't cargo automatically pick "0.22.1" ?

Reply:
I think you're right, unless for some reason 0.22.0 is in the lock file already or some package requests =0.22.0.

My intention was to require 0.22.1 as a minimum but be compatible with anything 0.22.* otherwise. That avoids any accidental issues and also documents that this is bog compatible with 0.22.0.

I have to say, I don't feel very confident about cargo versioning best practices, though. Do you think going with 0.22 is perferred here?